nupic/algorithms/anomaly.py

```diff
@@ -154,9 +161,33 @@ def compute(self, activeColumns, predictedColumns,
       else:
         score = 0.0
 
+    self._iteration += 1
+    if not self.isReady():
+      score = 0.5
```

Comment:
I think this makes sense. Maybe in CLAModel we can read this `burnin` value from TP and apply it to Anomaly.